src/components/Filters/Collections/FindCollections/FindCollections.js

```diff
@@ -1,59 +1,34 @@
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 
-import { Col, Row } from '@folio/stripes/components';
 import { Pluggable } from '@folio/stripes/core';
 
 const FindCollections = ({
   collectionIds,
   filterId,
   form,
   isEditable,
-  ...props
 }) => {
   const getSelectedCollections = (records) => {
     form.mutators.setCollection({}, records);
   };
 
-  const disableRecordCreation = true;
-  const buttonProps = { 'marginBottom0': true };
-
-  const pluggable =
+  return (
     <Pluggable
       aria-haspopup="true"
-      buttonProps={buttonProps}
       collectionIds={collectionIds}
       dataKey="collection"
-      disableRecordCreation={disableRecordCreation}
+      // plugin needs filterId for assigned filter
       filterId={filterId}
       id="clickable-find-collection"
       isEditable={isEditable}
-      marginTop0
-      onCloseModal={(modalProps) => {
-        modalProps.parentMutator.query.update({
-          query: '',
-          filters: '',
-          sort: 'name',
-        });
-      }}
       searchButtonStyle="default"
       searchLabel={<FormattedMessage id="ui-finc-select.plugin.buttonLabel.collection.add" />}
       selectRecordsModal={getSelectedCollections}
```

Comment:
Whats the `selectRecordsModal` prop used for?

Reply:
Thats needed for passing the IDs of the selected metadata-collections out and save them in the filter of the finc-select-app
https://github.com/folio-org/ui-plugin-find-finc-metadata-collection/blob/460d9fe8b2884e361e6be2bc1998057d10dbe234/src/CollectionSearch.js#L79
https://github.com/folio-org/ui-plugin-find-finc-metadata-collection/blob/460d9fe8b2884e361e6be2bc1998057d10dbe234/src/CollectionSearchModal.js#L25

Reply:
Ok. Got it. This is confusing tho, because `<CollectionSearch>` already defines a property `selectRecords` thats exactly indended for passing such a function.